github/table_github_repository_sbom.go

```diff
@@ -0,0 +1,103 @@
+package github
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableGitHubRepositorySbom() *plugin.Table {
+	return &plugin.Table{
+		Name:        "github_repository_sbom",
+		Description: "SBOM from a repository.",
+		List: &plugin.ListConfig{
+			KeyColumns: []*plugin.KeyColumn{
+				{
+					Name:    "repository_full_name",
+					Require: plugin.Required,
+				},
+			},
+			ShouldIgnoreError: isNotFoundError([]string{"404", "403"}),
+			Hydrate:           tableGitHubRepositorySbomList,
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "repository_full_name",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromQual("repository_full_name"),
+				Description: "The full name of the repository (login/repo-name).",
+			},
+			{
+				Name:        "spdx_id",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("SPDXID"),
+				Description: "The SPDX identifier for the SPDX document.",
+			},
+			{
+				Name:        "spdx_version",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("SPDXVersion"),
+				Description: "The version of the SPDX specification that this document conforms to.",
+			},
+			{
+				Name:        "creation_info",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("CreationInfo"),
+				Description: "The version of the SPDX specification that this document conforms to.",
+			},
+			{
+				Name:        "name",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
+				Description: "The name of the SPDX document.",
+			},
+			{
+				Name:        "data_license",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DataLicense"),
+				Description: "The license under which the SPDX document is licensed.",
+			},
+			{
+				Name:        "document_describes",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DocumentDescribes"),
```

Comment:
Do we need the transform for this column (and others)? Usually we only need these when the default transformation (https://github.com/turbot/steampipe-plugin-github/blob/main/github/plugin.go#L18) fails, usually when there's a field name with acronyms, e.g., `SPDXID`. I think we may only need these explicit transforms for the `spdx_id` and maybe the `spdx_version` columns.